pkg/models/translate.go

```diff
@@ -8,6 +8,16 @@ import (
 	"github.com/gofrs/uuid"
 )
 
+type ErrEditPrerequisiteFailed struct {
+	field    string
+	expected interface{}
+	actual   interface{}
+}
+
+func (e *ErrEditPrerequisiteFailed) Error() string {
+	return fmt.Sprintf("Expected %s to be “**%v**”, but was “**%v**”.", e.field, e.expected, e.actual)
```

Comment:
> Edit application failed:

> Prerequisite failed: Expected Details to be “text text text”, but was “****”.

Empty values cause the 'bold' markdown to not render.

Reply:
Thanks, fixed!